docs/api/extensions.md

```diff
@@ -78,6 +78,7 @@ The following methods of `chrome.runtime` are supported:
 - `chrome.runtime.getURL`
 - `chrome.runtime.connect`
 - `chrome.runtime.sendMessage`
+- `chrome.runtime.reload` (only supported in background page/script)
```

Comment:
I don't think we need to note that this is only supported in the background page, because that is the same as how it works in Chromium.

Reply:
Not exactly, for chromium `chrome.runtime.reload` is also supported in files accessed using `chrome-extension://` protocol. This is not exactly the case with an electron, I have an electron app where I load my chrome extension and then load an extension page URL, like `chrome-extension://<dynamic_extension_id>/test.html` but this page has no access for `chrome.runtime.reload`.

Here is a very basic chrome extension repo to use as an extension for testing: https://github.com/utkarsh-dixit/electron-extension-reload-test

<img width="1439" alt="Chrome_Extension_Runtime_Reload" src="https://user-images.githubusercontent.com/16796008/125681505-25cfaa98-441b-4be8-96b4-dfca665e650e.png">

Reply:
> @nornagon I tested it out on electron `v14.0.0-nightly` and `chrome.runtime.reload` is not supported. Below are the runtime api's that were available inside electron (i.e Object.keys(chrome.runtime))
> 
> `["id", "onMessage", "onConnect", "connect", "getManifest", "getURL", "sendMessage", "OnInstalledReason", "OnRestartRequiredReason", "PlatformArch", "PlatformNaclArch", "PlatformOs", "RequestUpdateCheckStatus"]`
> 
> Idk what this code is supposed to be for, but I am guessing it might be related to reloading extension on crash.

These were the keys I got when I ran `Object.keys(chrome.runtime)` inside a page loaded similarly like `chrome-extension://<dynamic_extension_id>/test.html`.

Reply:
It does seem to be available there:

![Screen Shot 2021-07-14 at 2 39 31 PM](https://user-images.githubusercontent.com/172800/125696615-660d9fa9-107b-4c31-a84a-b178706c7df8.png)

Reply:
Sorry my bad, I was using another one of my electron app to check if `runtime.reload` is available and there I found out I was also using [--disable-site-isolation-trials](https://www.chromium.org/_/rsrc/1521696303997/Home/chromium-security/site-isolation/site-isolation-flag-2.png) chromium flag. I understand that using this flag is not recommended and may cause potential security issues, but is the behaviour going on with `runtime.reload` expected or should it be mentioned explicitly in docs somewhere?

In case you want to take a look, this is the repo I have been using https://github.com/utkarsh-dixit/electron-extension-runtime-reload

PS: I have update the PR as per your suggestion.

![electron-flag-runtime-reload](https://user-images.githubusercontent.com/16796008/125701040-04289dc4-3a8b-4ac1-a15a-f6e52944b946.png)

Reply:
`--disable-site-isolation-trials` is not a mode we try very hard to support. It fundamentally changes the process model of Chromium and we make some assumptions about it. If you can show that Chromium itself handles that flag correctly in these circumstances, then file a bug on it. Though I wouldn't expect a speedy response :)